.github/workflows/CI.yml

```diff
@@ -57,6 +57,11 @@ jobs:
           version: ${{ matrix.version }}
           arch: ${{ matrix.arch }}
       - uses: julia-actions/cache@v1
+      # In MKL 2024, Intel dropped support for macOS.
+      # So, in CI, for the macOS jobs, we force MKL 2023 to be installed (instead of
+      # MKL 2024).
+      - run: julia .ci/macos_mkl_2023.jl
```

Comment:
Wouldn't `Pkg.add(; name="MKL_jll"; version="2023")` be enough?

Reply:
There are test dependencies, so in theory the `Pkg.test()` re-resolve can do whatever it wants.